.github/workflows/deployment-images.yaml

```diff
@@ -0,0 +1,109 @@
+          echo "::set-output name=repository::$repository"
+
+  set-version:
+    runs-on: ${{ inputs.runs-on }}
+    outputs:
+      version: ${{ steps.set-version.outputs.version }}
+    steps:
+      - id: set-version
+        run: |
+          short_sha=$(echo ${{ github.sha }} | head -c 7)
+          date=$(date -u +%Y%m%d)
+          version=$date-${{ github.run_number }}-$short_sha
+          echo "::set-output name=version::$version"
+
+  set-release-comment:
+    needs: [set-repository, set-version]
+    runs-on: ${{ inputs.runs-on }}
+    outputs:
+      release-comment: ${{ steps.set-release-comment.outputs.release-comment }}
+    steps:
+      - id: set-release-comment
+        run: |
+          repo=${{ needs.set-repository.outputs.repository }}
+          text="ghcr.io/${{ github.repository }}:${{ needs.set-version.outputs.version }}"
+          link="https://github.com/orgs/rentpath/packages/container/package/${repo}"
+          comment="Docker Image: [$text]($link)"
+          echo "::set-output name=release-comment::$comment"
+
+  deployment-image:
+    name: Build and Push the Deployment Image
+    needs: [set-release-comment, set-version]
+    runs-on: ${{ inputs.runs-on }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set /ops/version Build Information
+        run: |
+          echo "---" > ${{ inputs.build-information-path }}
+          echo "version: ${{ needs.set-version.outputs.version }}" >> ${{ inputs.build-information-path }}
+          echo "build_number: ${{ github.run_number }}" >> ${{ inputs.build-information-path }}
+          echo "git_commit: ${{ github.sha }}" >> ${{ inputs.build-information-path }}
+      - name: Login to GHCR
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ secrets.GHCR_USER_NAME }}
+          password: ${{ secrets.GHCR_USER_PAT }}
+      - uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: true
+          tags: ghcr.io/${{ github.repository }}:${{ needs.set-version.outputs.version }}
+      - name: Create GitHub Release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ needs.set-version.outputs.version }}
+          release_name: ${{ needs.set-version.outputs.version }}
+          body: '${{ needs.set-release-comment.outputs.release-comment }}'
+
+  deployment-test-image:
+    name: Build and Push the Deployment Test Image
+    if: inputs.deployment-test-image-docker-file-path != ''
+    needs: [set-version]
+    runs-on: ${{ inputs.runs-on }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Login to GHCR
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ secrets.GHCR_USER_NAME }}
+          password: ${{ secrets.GHCR_USER_PAT }}
+      - uses: docker/build-push-action@v2
+        with:
+          context: .
```

Comment:
Does this need to be the directory containing `${{ inputs.deployment-test-image-docker-file-path }}`?  It seems a bit odd to use a Dockerfile contained anywhere with `${{ inputs.deployment-test-image-docker-file-path }}` (as is the case in https://github.com/rentpath/seo_metadata_service/pull/214/files#diff-e063f8bc23d3a447083c4d63fd3417a4f53e304e38636e2e804f775dd97afb41) but then set the context to the repo root.

Reply:
Actually, I guess if we are going to hardcode this value, `.` is best. With the root directory set as the context all sub directories will be available.